blk/components/virt.c

```diff
@@ -254,6 +265,7 @@ static void handle_driver()
             }
         } else {
             // When more error conditions are added, this will need to be updated to a switch statement
+            // => shouldn't this pass along the exact status from the driver?
             err = blk_enqueue_resp(&h, SEEK_ERROR, drv_success_count, cli_data.cli_req_id);
```

Comment:
Yes, the status codes possible from the driver are the same as the virtualiser. Right now it does not matter since the only possible error is `SEEK_ERROR`, but feel free to add a commit/PR that passes whatever error it got from the driver.

Reply:
Thanks, could you put it in a separate commit though? Same PR is fine.